### DISCUSSION TITLE
Multiuser audiostream

### DISCUSSION THREAD

**Discussion by stsch83** [music-assistant/support#2718] [Q&A] [answered]
Hello, first of all I really like this extension. 
I am trying to use this extension for my whole family. All my children have their own speakers in their rooms and I want each child to see only their own speaker and not the speakers of the others. It should also be possible for all the children to listen to different music at the same time. I hope everyone can understand what I want. So I hope someone can give some advice on how to solve/configure the problems.
Br, Stefan

**Comment by OzGav** [comment, marked as answer]
Everyone can already to listen to different music at the same time. Currently you cannot easily hide speakers from others. To do so you would have to do some clever stuff in HA, for example, https://github.com/orgs/music-assistant/discussions/2601
